students/wachulec_maria/lesson_06_dicts_tuples_sets_args_kwargs/frequency_analysis.py

```diff
@@ -0,0 +1,14 @@
+def frequency_analysis():
+    words_frequency = {}
+    for i in range(int(input())):
+        words = input().split()
+        for j in words:
+            words_frequency.setdefault(j, 0)
+            words_frequency[j] += 1
+    frequency_list = sorted(sorted(list(words_frequency.items()),
```

Comment:
good.
you can also try out tuple sorting properties:
`frequency_list = sorted([(-count, word) for word, count in words_frequency.items()])`